patching/plugin-commands-patching/src/patchRemove.ts

```diff
@@ -72,7 +72,11 @@ export async function handler (opts: PatchRemoveCommandOptions, params: string[]
       }
     } catch {}
   }))
-
+  Object.entries(patchedDependencies).forEach(([key, value]) => {
+    if (path.isAbsolute(value)) {
+      patchedDependencies[key] = path.relative(opts.dir, value)
```

Comment:
It should be relative to `opts.workspaceDir ?? opts.rootProjectManifestDir`, not to `opts.dir`.

Reply:
Updated.